air-script/tests/bitwise/bitwise.air

```diff
@@ -0,0 +1,56 @@
+def BitwiseAir
+
+public_inputs:
+    stack_inputs: [16]
+
+trace_columns:
+    main: [s, a, b, a0, a1, a2, a3, b0, b1, b2, b3, zp, z, dummy]
+
+periodic_columns:
+    k0: [1, 0, 0, 0, 0, 0, 0, 0]
+    k1: [1, 1, 1, 1, 1, 1, 1, 0]
+
+boundary_constraints:
+    # This is a dummy trace column to satisfy requirement of atleast one boundary constraint.
+    enf dummy.first = 0
+
+transition_constraints:
+    # Enforce that selector must be binary
+    enf s^2 - s = 0
+
+    # Enforce that selector should stay the same throughout
+    enf k1 * (s' - s) = 0
+
+    # Enforce that input is decomposed into valid bits
+    enf a0^2 - a0 = 0
+    enf b0^2 - b0 = 0
+    enf a1^2 - a1 = 0
+    enf b1^2 - b1 = 0
+    enf a2^2 - a2 = 0
+    enf b2^2 - b2 = 0
+    enf a3^2 - a3 = 0
+    enf b3^2 - b3 = 0
+
+    # Enforce that the values in the columns a and b are exactly equal to the aggregation of binary
+    # values contained in the individual bit columns.
```

Comment:
nit: I would prefer if the comment specifies that these constraints are only applied in the first row. For example, the existing comment in our implementation is "The values in column a in the first row should be the aggregation of the decomposed bit columns a0..a3." (with an analogous comment for b)